### PR TITLE
Adding a documentation section in the README

### PR DESCRIPTION
**Pull request by minhtuev**
Adding instructions for documentation in the README doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,18 @@ pytest .
 
 After making changes and all tests are passing locally, commit the code to your fork and issue a PR against the [cisco/mindmeld](https://github.com/cisco/mindmeld) repo.
 
+## Documentation
+You can view the documentation for the latest stable release on [mindmeld.com/docs](http://mindmeld.com/docs) and the documentation for the latest change off the master change on our [Github Page](https://cisco.github.io/mindmeld/).
```
Comment (serapio) [inline README.md:108]:
```suggestion
You can view the documentation for the latest stable release on [mindmeld.com/docs](http://mindmeld.com/docs) and the documentation for the latest master branch version on our [Github Page](https://cisco.github.io/mindmeld/).
```

File: README.md [hunk 2]
```diff
@@ -104,6 +104,18 @@ pytest .
...
+
+You can contribute to the documentation under `sources` directory.
```
Comment (serapio) [inline README.md:110]:
```suggestion
You can contribute to the documentation under the `source` directory.
```

File: README.md (hunk 1 above)
Comment (minhtuev) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (minhtuev) [reply README.md]: Done

**Pull request merged**